src/__tests__/index-test.tsx

```diff
@@ -265,6 +270,27 @@ describe('redux-react-hook', () => {
       expect(renderCount).toBe(2);
     });
 
+    it('renders once if have multiple useMappedState', () => {
```

Comment:
this seems to be the real reason to use `useLayoutEffect`, so why isn't this in the comment?

Reply:
updated comment